lib/workers/repository/init/index.ts

```diff
@@ -26,8 +26,9 @@ export async function initRepo(
   config_: RenovateConfig
 ): Promise<RenovateConfig> {
   let config: RenovateConfig = initializeConfig(config_);
-  await initializeCaches(config);
+  await resetCaches();
   config = await initApis(config);
+  await initializeCaches(config);
```

Comment:
You may ask why not just move `initializeCaches()` call? The reason is that I wanted to reset caches from previous run, i.e. prevent data leaks between repositories.